find/less/__init__.py

```diff
@@ -0,0 +1,87 @@
+import check50
+import check50.c
+
+@check50.check()
+def exists():
+    """helpers.c exists."""
+    check50.exists("helpers.c")
+
+@check50.check(exists)
+def compiles():
+    """helpers.c compiles."""
+    check50.include("helpers.c")
```

Comment:
`include` is used to include additional files not submitted by the student. `helpers.c` is submitted by the student. `helpers.h`, `sort.c`, and `find.c` do need to be included though via

```
check50.include("helpers.h", "find.c", "sort.c")
```